test/specs/integration.spec.js

```diff
@@ -36,26 +36,23 @@ describe('Integration tests', function() {
     });
   }
 
-  beforeEach(function() {
+  beforeEach(function(done) {
     // Make sure we're connected to Firebase.  This can take a while on slow
     // connections.
     var ref = new Firebase('https://firepad-test.firebaseio-demo.com/.info/connected');
     var connected = false;
     var listener = ref.on('value', function(s) {
-      connected = s.val() == true;
-    });
-
-    waitsFor(function() { return connected; }, 'connected', 10000);
-
-    runs(function() {
-      ref.off('value', listener);
+      if(s.val() == true){
```

Comment:
Can you add a space before the open brace?

(sorry, I should probably set up linting or something)